R/CohortDefinition.R

```diff
@@ -281,3 +281,8 @@ getCohortSql <- function(cohortDefinition, baseUrl, generateStats = TRUE) {
     stop()
   }
 }
+
+#' @export
+print.cohort <- function(x, ...) {
```

Comment:
why make it exportable

Reply:
You're right. I removed the export tag.

Reply:
Actually I removed this change altogether since it is not relevant to this PR. Adding generic methods for ROhdsiWebApi objects is discussed in https://github.com/OHDSI/ROhdsiWebApi/pull/150